### ISSUE TITLE
封装VMamba调用

### ISSUE THREAD

**Issue by MPCheng-ZW** [MzeroMiko/VMamba#261]
您好！

请问有可以封装好的VMamba可以调用吗？

类似于mamba这种：
import torch
from mamba_ssm import Mamba

batch, length, dim = 2, 64, 16
x = torch.randn(batch, length, dim).to("cuda")
model = Mamba(
    # This module uses roughly 3 * expand * d_model^2 parameters
    d_model=dim, # Model dimension d_model
    d_state=16,  # SSM state expansion factor
    d_conv=4,    # Local convolution width
    expand=2,    # Block expansion factor
).to("cuda")
y = model(x)
assert y.shape == x.shape

谢谢~

**Comment by MzeroMiko** [comment]
There's no pip packages released, and you have to install selective-scan kernel on your own. But we will release the pre-build wheels in the release page soon.

**Comment by MzeroMiko** [comment]
## 🔥 use VMamba with only ***one file*** and in ***fewest steps*** !
```bash
conda create -n vmamba python=3.10
pip install torch==2.2 torchvision torchaudio triton pytest chardet yacs termcolor fvcore seaborn packaging ninja einops numpy==1.24.4 timm==0.4.12
pip install https://github.com/state-spaces/mamba/releases/download/v2.2.4/mamba_ssm-2.2.4+cu12torch2.2cxx11abiTRUE-cp310-cp310-linux_x86_64.whl
python vmamba.py
```

**Comment by Li-yachuan** [comment]
> ## 🔥 use VMamba with only _**one file**_ and in _**fewest steps**_ !
> conda create -n vmamba python=3.10
> pip install torch==2.2 torchvision torchaudio triton pytest chardet yacs termcolor fvcore seaborn packaging ninja einops numpy==1.24.4 timm==0.4.12
> pip install https://github.com/state-spaces/mamba/releases/download/v2.2.4/mamba_ssm-2.2.4+cu12torch2.2cxx11abiTRUE-cp310-cp310-linux_x86_64.whl
> python vmamba.py

I feel like the packaged version runs a lot slower, is this just my illusion?
